quay_search.py

```diff
@@ -0,0 +1,133 @@
+#!/home/wolffj/miniconda2/bin/python2.7
```

Comment:
Will only work on your computer. For portability, please use `#!/usr/bin/env python2`